src/main/java/input/Gamepad.java

```diff
@@ -4,6 +4,10 @@
 
 // https://github.com/LWJGL/lwjgl3-wiki/wiki/2.6.3-Input-handling-with-GLFW#joystick-input
 
+/**
+ * <h1>Azurite</h1>
+ * Abstraction over LWJGL specification of gamepad buttons.
```

Comment:
More specifically, the API we are making use of is provided by GLFW, not just LWGJL.